src/components/input/input.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './input.css';
+
+const Input = ({inputValue, handleChange, placeholder, styleclass}) => {
+    return (
+    <input
+    value={inputValue}
+    onChange={handleChange}
+    placeholder={placeholder}
+    className={styleclass} />
```

Comment:
Did we really need to pass the `styleclass` from the parent component?

Since the styles are defined inside the component folder and its not an element that may have different type of styles why not simply use the styles in the component?